### ISSUE TITLE
mintReport opens "page not found" for github issues page

### ISSUE THREAD

**Issue by ptandler** [linuxmint/mintreport#57]
I have a crash report (see below). When I select a crash report and click on "issue tracker" ("Fehlerverfolgungssystem" in German ;-)) in mintReport, it opens https://github.com/linuxmint/libxapp1/issues - but this pages does not exist.

Beside the fact that this should be corrected: Which issue tracker is the correct one for this crash?

```
===================================================================
 Info                                                              
===================================================================
           PID: 2131 (xapp-sn-watcher)
           UID: 1000 (ptandler)
           GID: 1000 (ptandler)
        Signal: 11 (SEGV)
     Timestamp: Mon 2020-08-10 22:32:57 CEST (42min left)
  Command Line: /usr/libexec/xapps/sn-watcher/xapp-sn-watcher
    Executable: /usr/libexec/xapps/sn-watcher/xapp-sn-watcher
 Control Group: /user.slice/user-1000.slice/session-c2.scope
```

**Comment by okaestne** [comment]
Moin ;)
The correct link would be https://github.com/linuxmint/xapp/issues.

The link is generated by mintreport from the name of the package which provides the crashed binary.
https://github.com/linuxmint/mintreport/blob/master/usr/lib/linuxmint/mintreport/app.py#L511

**Comment by ptandler** [comment]
So this would require some mapping of package names to github project names for those cases where this isn't identical ... right?

**Comment by okaestne** [comment]
Yeah and this isn't a really clean design. I would suggest using the "Bugs" section in the Debian control file instead, but we need to add the URLs in all Mint packages.
